src/app/pages/common/ix/components/ix-combobox/ix-combobox.component.ts

```diff
@@ -19,13 +26,17 @@ import { Option } from 'app/interfaces/option.interface';
     },
   ],
 })
-export class IxComboboxComponent implements ControlValueAccessor, OnChanges {
+export class IxComboboxComponent implements ControlValueAccessor, OnChanges, OnInit {
   @Input() label: string;
   @Input() hint: string;
   @Input() required: boolean;
   @Input() tooltip: string;
   @Input() options: Observable<Option[]>;
   @ViewChild('ixInput') inputElementRef: ElementRef;
+  @ViewChild('auto') autoCompleteRef: MatAutocomplete;
+  @ViewChild(MatAutocompleteTrigger) autocompleteTrigger: MatAutocompleteTrigger;
+  @Output() scrollEnd: EventEmitter<string> = new EventEmitter<string>();
+  placeholder = T('Search');
```

Comment:
also, it probably makes sense to add `@Input()` decorator to make it customizable

Reply:
The idea is that keeping the placeholder fixed to `Search` would indicate that the field is searchable. Otherwise, it might seem like just another text input.